finetuning/llama/inference.py

```diff
@@ -0,0 +1,77 @@
+# inference.py
```

Comment:
add comment that shows command to deploy to beam

Reply:
I added 

"Deploy to beam by running `$ python inference.py` in the terminal"